docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -22,10 +22,10 @@ NOTE: KQL autocomplete for `.alerts-security.alerts-*` indices is available on t
 `Detection alerts` is selected.
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
-* View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, then click on a rule name in the All rules table to view the alerts associated with the rule.
+* View detection alerts generated by a specific rule. Go to *Detect* -> *Rules*, then select a rule name in the All rules table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule (including alerts from any previous or deleted revision of that rule).
 * Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the *Count* table. For example, if you select `kibana.alert.rule.name`, the table displays the number of alerts generated by each rule.
 * Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. For example, if you select `kibana.alert.rule.name`, the histogram displays the total alert counts by rule name.
-* Hover over a value in the data grid to display available inline actions, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand button to open a full context menu of options, including *Show top values* and *Copy to Clipboard*. The options available vary depending on the type of data.
+* Hover over a value in the data grid to display available inline actions, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand button to open a full context menu of options, including *Show top values*, *Copy to Clipboard*, and *View rule details*. The options available vary depending on the type of data.
```

Comment:
```suggestion
* Hover over a value in the data grid to display available inline actions, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand button to open a full context menu of options, including *Show top values*, *Copy to Clipboard*, and *View rule details*. The available options vary depending on the type of data.
```
I'm thinking it's either "The options that are available" or "The available options".

Reply:
Agree!